app/helpers/component_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ComponentHelper
+  def add_option(option, output = nil)
```

Comment:
Ok, thanks for the clearification.

I'm more so asking that @samcdavid attach his comments at the end of a method (if the comment is regarding the whole method) rather than the first line so that it is easier to read in the conversation pane on a github PR.